apps/authentication-service/cypress/e2e/clients.cy.ts

```diff
@@ -1,14 +1,21 @@
 import { Keys } from "@/resources/clients/clientsModel";
 import { PORT } from "@/shared/consts";
+import { WidgetHostPermissions } from "@/shared/enums";
+import { ClientGrant } from "auth0";
 
 const USER_ID: string = "auth0|667c3d0c90b963e3671f411e";
 
 describe("Client API", () => {
   let accessToken: string;
   let accessTokenBasic: string;
+  let accessTokenM2M: string;
   let newClientId: string;
   let newClientSecret: string;
+
   const keysUrl = `http://localhost:${PORT}/v1/clients/keys`;
+  const auth0BaseUrl = Cypress.env("AUTH0_DOMAIN") as string;
+  const auth0ClientGrantsUrl = `${auth0BaseUrl}/api/v2/client-grants`;
+  const clientScope = `${Object.values(WidgetHostPermissions).join(" ")}`;
 
   const getTokens = () => {
     cy.window()
```

Comment:
There's a lot of duplication in this function

Reply:
I cleaned that up a bunch. Let me know what you think.